docs/performance/app_open_performance.md

```diff
@@ -55,24 +55,23 @@ To further understand why app open times are slow it is important to find out ex
 
 The engine goes through many stages when a user wants to attach to an app, and some of those stages will depend on whether or not the app is already loaded into memory. The following progress report shows a typical case of a "cold open", where the app has not already been loaded into memory:
 
-![Progress_report_cold_open.png](images/Progress_report_cold_open.png)
+![Progress_report_cold_open.png](images/Progress_report_cold_open_highlighted.png)
```

Comment:
Suggesting to have image number to easier be able to explain the comparison later.

Example (or something similar):
```suggestion
![Progress_report_cold_open.png](images/Progress_report_cold_open_highlighted.png)
(image 1.1)
```